can-value.md

```diff
@@ -101,31 +77,21 @@ Use `can-value` when you need an observable that can get or set a property on an
 In the example below, we use [can-value.bind] to get an observable that
 can get _and_ set `outer.inner.key`:
 
-```js
-import DefineMap from "can-define/map/map";
-import value from "can-value";
-
-const outer = new DefineMap({
-  inner: {
-    key: "hello"
-  }
-});
-
-const keyObservable = value.bind(outer, "inner.key");
-```
+@sourceref ./doc/examples/observableObject.js
+@codepen
+@highlight 9,only
 
 Now if we read `keyObservable.value`, we get the value at `outer.inner.key`:
 
-```js
-keyObservable.value; // is "hello"
-```
+@sourceref ./doc/examples/observableObject.js
+@codepen
+@highlight 10,only
 
 We can also set `keyObservable.value` to change the value at `outer.inner.key`:
 
-```js
-keyObservable.value = "aloha";
-// Now outer.inner.key === "aloha"
-```
+@sourceref ./doc/examples/observableObject.js
+@codepen
+@highlight 12,only
```

Comment:
This page seems to be missing why someone would use `.from` vs `.to` vs `.bind` ...

Typically these are used with dynamic components.  We should make sure this is known.  You might not be able to write this up, but create an issue about it for it to be done later.

Reply:
Opened #22